kotlin-jvm/src/main/kotlin/playground/FuelHttp.kt

```diff
@@ -0,0 +1,19 @@
+package playground
```

Comment:
Pease
- put the file in its own package
- call your function from the main file `_main.kt`

```
@file:Suppress("PackageDirectoryMismatch")
package playground.fuel
```